package.json

```diff
@@ -13,10 +13,10 @@
   "devDependencies": {
     "bower": "1.3.8",
     "chai": "1.10.0",
-    "jshint": "2.5.2",
+    "jshint": "2.8.0",
```

Comment:
Shouldnt you define min-versions instead of fixed ones, e.g. Using caret operator `^`?